packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -1452,6 +1452,25 @@ const tests = {
         }
       `,
     },
+    // TODO
+    {
+      code: normalizeIndent`
```

Comment:
```suggestion
      skip: true,
      code: normalizeIndent`
```

if I'm not wrong you could do this to skip

Reply:
you're right, thanks!